packages/provider/src/context.ts

```diff
@@ -4,7 +4,8 @@ export const WalletContext: ArcadeumContext = {
   factory: '0x52f0F4258c69415567b21dfF085C3fd5505D5155',
   mainModule: '0x57bA6Eb1ed6821Db7bC26C6714f093E2BCbea40F',
   mainModuleUpgradable: '0xF52136A04057d889CeBf9bCc2F3142965AeABc54',
-  guestModule: '0xe076ad01F1eb18A8eF20bB64003DA4810a429a32'
+  guestModule: '0xe076ad01F1eb18A8eF20bB64003DA4810a429a32',
+  requireUtils: '0xd46566c23ae549fc14abfcfcbc130e3ea7e63f82'
```

Comment:
what is this contract about btw?

Reply:
It's a very small contract that implements the functionality for "expirable" transactions and "linked" transactions. It has methods with only a "require" in them, so the wallet can include txs to those methods with `revertOnError: true` and with that enforce those conditions on the transaction batch.